README.md

```diff
@@ -55,10 +55,17 @@ Scans PHP and JavaScript files, as well as theme stylesheets for translatable st
 	[--ignore-domain]
 		Ignore the text domain completely and extract strings with any text domain.
 
-	[--merge[=<file>]]
-		Existing POT file file whose content should be merged with the extracted strings.
+	[--merge[=<files>]]
+		One or more existing POT files whose contents should be merged with the extracted strings.
```

Comment:
The syntax for multiple files should be documented.

Reply:
Do you mean documented like `<file1,file2>` or as an example in the argument description?

Reply:
Not sure what the best way is, as you have multiple arguments with the same issue.

Maybe a small adjective like `comma-separated` and then a general note in the command description might be clearer?

```
	[--merge[=<file[,file2,...]>]]
		Comma-separated list of POT files whose contents should be merged with the extracted strings.
```

Something like that, maybe? Unsure about the notation...

Reply:
Turns out something like `[--merge[=<file[,file2,...]>]]` isn't valid.

I'll stick with the description though.